src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -15,7 +16,8 @@ import API from '../../api';
 import Cache from '../../util/Cache';
 import Internationalize from '../Internationalize';
 import { DEFAULT_HOSTNAME_API, CLIENT_NAME_CONTENT_SIDEBAR } from '../../constants';
-import type { AccessStats, Token, BoxItem, StringMap, FileVersions } from '../../flowTypes';
+import intlMessages from '../messages';
```

Comment:
Normally in other files we just do `import messages from '../messages';`

Reply:
I noticed this, but in the render method it takes in messages as a prop, and an eslint error is thrown since its declared in upper scope. Seems there are 3 options: 
1) my current approach
2) change the prop name in the destructuring of the props in the render method
3) eslint ignore

Maybe approach 2 is better, for consistency?